bin/gui-terminal

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+python3 ../ganga/GangaGUI/gui/webterm.py </dev/null &>/dev/null &
```

Comment:
If you want to be able to kill the terminal that you start this script from and keep the GUI running, you will need to disown the process as well. Also some thought should be given to how the server process eventually will stop.

Reply:
I will look into this

Reply:
actually </dev/null &>/dev/null & provides same functionality as disown process, this will also prevent unwanted logs to be printed.

Reply:
No, I do not think so. When you do this, the process group of the child process stays the same (so killing the parent will still kill the child). So I think you need to do the above AND disown the process.

Reply:
Added the disown command